Competition2018/src/main/java/competition/subsystems/offboard/commands/AcquireVisibleCubeCommand.java

```diff
@@ -0,0 +1,20 @@
+package competition.subsystems.offboard.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.drive.DriveSubsystem;
+import competition.subsystems.offboard.OffboardCommunicationPacket;
+import competition.subsystems.offboard.OffboardInterfaceSubsystem;
+
+public class AcquireVisibleCubeCommand extends OffboardProcessingWithDriveCommand {
+
+    @Inject
+    protected AcquireVisibleCubeCommand(OffboardInterfaceSubsystem offboardSubsystem, DriveSubsystem driveSubsystem) {
+        super(0x01, offboardSubsystem, driveSubsystem);
```

Comment:
Should this be replaced with OffboardCommsConstants.PACKET_TYPE_WHEEL_ODOM  ?

Reply:
The value here is the command ID, which is independent from packet types. I didn't give it its own constant variable because it shouldn't ever need to be referenced by other components on this end.